shoop/simple_cms/admin_module/views.py

```diff
@@ -128,7 +128,10 @@ def save_form(self, form):
 class PageListView(PicotableListView):
     model = Page
     columns = [
-        Column("title", _(u"Title"), linked=True, filter_config=TextFilter(operator="startswith")),
+        Column(
+            "title", _(u"Title"), sort_field="translations__title", display="title", linked=True,
+            filter_config=TextFilter(operator="startswith")
+        ),
```

Comment:
Adding display="title" is not really needed, since default display is the same as id which is also "title" here, but that doesn't do any harm either.